File: ass47/program3.c
/*
3. Write a recursive program which accept string from user and count number of small characters. 
Input : HElloWOrlD 
Output : 5
*/
#include<stdio.h>
int Display(char *str)
{
    static int i = 0;
    if(*str!='\0')
    {   
        if((*str>='a')&&(*str<='z'))
        {
            i++;
        }
        str++;
        Display(str);
    }
    return i;
}

int main()
{
    int iRet = 0;
    char arr[20];
    printf("Enter the string\n");
    scanf( "%s",arr);

    iRet = Display(arr);
    printf("%d",iRet);
    return 0;
}

File: ass55/Program3.java
/*
3. Write java program which accept N numbers from user and accept one another number as NO , return index of last occurrence of that NO. 
Input :  N :   6 
  NO:  66 
  Elements : 85 66 3 66 93 88  
Output : 3 
Input :  N :   6 
  NO:  93 
  Elements : 85 66 3 66 93 88  
Output : 4 
Input :  N :   6 
  NO:  12 
  Elements : 85 11 3 15 11 111  
Output : -1
*/
import java.lang.*;
import java.util.*;
class ArrayDemo
{
    void Even(int arr[],int value)
    {
        int iSum1 = 0,iPos = 0;
        for(int i=0; i<arr.length; i++)
        {
            if(arr[i]==value)
            {
               iPos = i;
            }   
        }
        System.out.println("index is : " +iPos);
    }
}
class Program3
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Element");
        int size = sobj.nextInt();
        int value = sobj.nextInt();
        int arr[] = new int[size];
        System.out.println("Enter the Array elements");
        for(int i=0; i<arr.length; i++)
        {
            arr[i] = sobj.nextInt();
        } 
        ArrayDemo dobj = new ArrayDemo();
        dobj.Even(arr,value);
    }
}